core/domain/state_domain.py

```diff
@@ -3710,7 +3710,7 @@ def validate(
                 allowed to be unspecified.
             tagged_skill_misconception_id_required: bool. The 'tagged_skill_
                 misconception_id' is required or not.
-            strict: bool. Tells if the exploration is strict or not.
+            strict: bool. Tells if the entity is strict or not.
```

Comment:
Actually, this should be "Tells if the validation is strict or not. Validation should be strict for all published entities, i.e. those that are viewable by a learner. It can be non-strict for entities that are only viewable by lesson creators."

(Rationale: it's the validation that's strict, not the entity; also, there's not much point defining "strict" in terms of "strict".)

Reply:
Ah!, right. Updated.
@seanlip PTAL